test/suite/test_oligarch06.py

```diff
@@ -37,32 +37,26 @@ class test_oligarch06(wttest.WiredTigerTestCase):
 
     # conn_config = 'log=(enabled),verbose=[oligarch:5]'
     conn_base_config = 'oligarch_log=(enabled),statistics=(all),statistics_log=(wait=1,json=true,on_close=true),' \
-                     + 'disaggregated=(stable_prefix=.,storage_source=dir_store),'
+                     + 'disaggregated=(stable_prefix=.,page_log=palm),'
     conn_config = conn_base_config + 'oligarch=(role="leader")'
 
     # TODO do Python tests expect a field named uri?
     uri = "oligarch:test_oligarch06"
 
-    # Load the directory store extension, which has object storage support
+    # Load the page log extension, which has object storage support
     def conn_extensions(self, extlist):
         if os.name == 'nt':
             extlist.skip_if_missing = True
-        extlist.extension('storage_sources', 'dir_store')
+        extlist.extension('page_log', 'palm')
         self.pr(f"{extlist=}")
 
     # Custom test case setup
     def early_setup(self):
-        # FIXME: This shouldn't take an absolute path
-        os.mkdir('foo') # Hard coded to match library for now.
-        os.mkdir('bar') # Hard coded to match library for now.
         os.mkdir('follower')
-        os.mkdir('follower/foo')
-        os.mkdir('follower/bar')
-
-        # TODO shouldn't need these - work around code in __wt_open_fs that handles
-        # non fixed-location files
-        os.mkdir('follower/foo/follower')
-        os.mkdir('follower/bar/follower')
+        # Create the home directory for the PALM k/v store, and share it with the follower.
+        os.mkdir('kv_home')
+        os.symlink('../kv_home', 'follower/kv_home', target_is_directory=True)
+        os.system('echo setup; ls -l . ./follower')
```

Comment:
Remove this line. Also in `test_oligarch07`.

Reply:
@pmacko86 which line should be removed? The symlink setup is needed to connect the two kv_homes.

Reply:
I mean the line with "echo" and "ls".

Reply:
fixed.